### ISSUE TITLE
Improve fail over between exFAT and FAT32

### ISSUE THREAD

**Issue by Dees-Troy** [TeamWin/Team-Win-Recovery-Project#92]
Sometimes blkid fails to detect vfat so we need to improve the fail-over between exFAT and FAT32.

http://tinyw.in/prrn

**Comment by n0d3** [comment]
Don't know how clean or direct way this is, but
I'm currently using dumpexfat to detect if the fs-type of storage is exfat. 
In Check_FS_Type() before running blkid, I check the exit code of dumpexfat.

```
// ...
string Command, result;
if (Mount_Point == "/sdcard" || Mount_Point == "/external" || Mount_Point == "/external_sd")
{ 
    Command = "dumpexfat " + Actual_Block_Device + "&> /dev/null";
    if (TWFunc::Exec_Cmd(Command, result) == 0)
    {
        Current_File_System = "exfat";
        return;
    }
}
Command = "blkid " + Actual_Block_Device;
TWFunc::Exec_Cmd(Command, result);
// ...
```

**Comment by Dees-Troy** [comment]
I appreciate the suggestion, n0d3, but we decided to check the result from exfat-fuse when mounting versus having to build and include another binary. It's less code to maintain going forward this way and might help keep the final recovery image smaller.

We fixed this in 2.4.1.0
